builtin/providers/aws/resource_aws_elb.go

```diff
@@ -371,11 +386,47 @@ func resourceAwsElbUpdate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if d.HasChange("idle_timeout") {
+		log.Println("[INFO] inside modify attributes")
+		attrs := elb.ModifyLoadBalancerAttributesInput{
+			LoadBalancerName: aws.String(d.Get("name").(string)),
+			LoadBalancerAttributes: &elb.LoadBalancerAttributes{
+				ConnectionSettings: &elb.ConnectionSettings{
+					IdleTimeout: aws.Integer(d.Get("idle_timeout").(int)),
+				},
+			},
+		}
+		_, err := elbconn.ModifyLoadBalancerAttributes(&attrs)
+		if err != nil {
+			return fmt.Errorf("Failure configuring idle timeout: %s", err)
+		}
+		d.SetPartial("idle_timeout")
+	}
+
+	if d.HasChange("connection_draining") || d.HasChange("connection_draining_timeout") {
+		log.Println("[INFO] inside modify attributes")
+		attrs := elb.ModifyLoadBalancerAttributesInput{
+			LoadBalancerName: aws.String(d.Get("name").(string)),
+			LoadBalancerAttributes: &elb.LoadBalancerAttributes{
+				ConnectionDraining: &elb.ConnectionDraining{
+					Enabled: aws.Boolean(d.Get("connection_draining").(bool)),
+					Timeout: aws.Integer(d.Get("connection_draining_timeout").(int)),
+				},
+			},
+		}
+		_, err := elbconn.ModifyLoadBalancerAttributes(&attrs)
+		if err != nil {
+			return fmt.Errorf("Failure configuring connection draining: %s", err)
+		}
+		d.SetPartial("connection_draining")
+		d.SetPartial("connection_draining_timeout")
+	}
+
```

Comment:
I wasn't sure on how to handle this, cross_zone_load_balancing does the same. I could do it all in one request if thats preferable. I order to do so I would just see if any of the ELB attributes supported keys are changed and then compose the call. Let me know if you want me to go that route and I will make the change.